### ISSUE TITLE
Input of fit() and return value of predict_proba() method

### ISSUE THREAD

**Issue by Gedevan-Aleksizde** [ibayer/fastFM#152]
I try to have `FMClassification` class integrated with scikit-learn classes (e.g. `sklearn.metrics`, `sklearn.pipe` ). However the `fit()` method permits  only {-1, 1} for the binary labels and  `predict_proba()` method doesn't return the regular scikit-learn-like format, `(n_samples, n_classes)` array, but returns `(n_samples, )` array. Now I'm using a simple and imperfect wrapper class on my own, but  I want to know is there any reason so, and if you have a plan to change?
